lib/assessment.rb

```diff
@@ -5,7 +5,10 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    longest = %w{ cat sheep bear }.inject do |memo, word|
+    memo.length > word.length ? memo : word
+    end
+    longest
```

Comment:
Your approach is generally correct - however, we want to do operate on the collection `enum`, not just any collection that we might pick. 

Additionally, inject will already return the last result, so the final line there is unnecessary.